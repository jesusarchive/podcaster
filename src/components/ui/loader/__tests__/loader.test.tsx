// loader component tests using react-testing-library

import { render } from '@testing-library/react';
import React from 'react';

import Loader from '..';

describe('<Loader />', () => {
  it('should render the loader component', () => {
    const { container } = render(<Loader />);

    expect(container).toMatchSnapshot();
  });
});
