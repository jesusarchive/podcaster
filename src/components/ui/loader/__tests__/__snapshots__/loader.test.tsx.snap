// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should render the loader component 1`] = `
<div>
  <div
    class="loader"
  >
    <div />
    <div />
  </div>
</div>
`;
